src/features/chat/ChatMessage.tsx

```diff
@@ -0,0 +1,66 @@
+import React /*, { CSSProperties }*/ from 'react'
+import { chatMsgDatetime } from '../../common/utils/format'
+import cn from 'classnames'
+import styles from './Chat.module.css'
+
+export const DIR_SENT = 0
+export const DIR_RECV = 1
+
+export interface ChatMessageProps {
+  text?: string
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  sender: any
+  date: Date
+  unread: boolean
+  direction: number // incoming (1) / outcoming (0)
```

Comment:
Use enum instead of DIR_SENT and DIR_RECV and then no need for comment. And typescript could restrict it to be 0 or 1 only, currently it can be any number.